components/buttons.tsx

```diff
@@ -84,6 +86,181 @@ export const ImageButton = styled<
   padding: 1px;
 `
 
+export const TextButton = ({
+  label,
+  className
+}: { label: string; className?: string } & ButtonProps) => {
+  return (
+    <Button variant="link" className={`${className}`}>
+      {label}
+    </Button>
+  )
+}
+
+type FillButtonProps = ButtonProps & {
+  label?: string
+  Icon?: ReactNode
+}
+
+export const FillButton = ({
+  variant = "primary",
+  Icon,
+  className,
+  label,
+  ...rest
+}: FillButtonProps) => {
+  return (
+    <Button
+      variant={variant}
+      type="submit"
```

Comment:
when this button is type="submit" and an instance of the button is used in the Profile Settings Modal to save, the modal state isn't being saved for some reason.

when I change the type="button", then the Profile Settings Modal save button properly saves.

because FillButton gets used in a few other places (for Following/Unfollowing), I tried to test type="button" under those circumstances and it seemed to work, but it'd be good to have the author double check that I haven't missed a usage.